kernel/src/fs/exfat/inode.rs

```diff
@@ -1275,8 +1276,11 @@ impl Inode for ExfatInode {
         for _ in Bid::from_offset(read_off)..Bid::from_offset(read_off + read_len) {
             let physical_bid =
                 Bid::from_offset(cur_cluster.cluster_id() as usize * cluster_size + cur_offset);
-            inner.fs().block_device().read_block(physical_bid, &frame)?;
-            frame.read(0, writer).unwrap();
+            inner
+                .fs()
```

Comment:
Why not move line 1280 up to 1279

Reply:
I believe it's `cargo fmt`'s effort since the rest call chain is long. Nothing we can do on the formatting thing.